com.albiononline.AlbionOnline.json

```diff
@@ -0,0 +1,82 @@
+{
+    "app-id": "com.albiononline.AlbionOnline",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "albion-online",
+    "tags": ["proprietary"],
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
```

Comment:
Lines 9-12 are one space deeper than 13-14. I usually autoformat manifests with jq like this: `cat manifest.json | jq -M . > tmp.json` and if nothing exploded, replace the original file with the new one.